tests/mol-audio_exceeding_clipend/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:creator>Marisa DeMeglio</dc:creator>
+    <dc:date>2022-02-10</dc:date>
+    <dc:description>Tests whether the Reading System ignores the value of the clipEnd attribute in the smil file's 'par' element if it exceeds the length of the audio file, and moves to the next 'par' step immediately at the end that audio content</dc:description>
+    <dc:identifier id="pub-id">mol-exceeding_clipend</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Ignoring clipEnd attribute if it exceeds track length</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">may</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#mol-audio-exceeding-clipEnd</meta>
```

Comment:
I think this is unsubmitted, but for consistency with existing spec anchors (and the naming of this test), I think this anchor should be -clipend, not -clipEnd.